server/utils/activity/upsertActivity.ts

```diff
@@ -235,6 +235,9 @@ function countTimeRange(
           log.endMs < range.endMs &&
           log.endMs - range.startMs > ACTIVITY_COUNT_INTERVAL_THRESHOLD_MS)
       ) {
+        if (range.count === undefined) {
+          range.count = 0;
+        }
```

Comment:
一瞬だけ検討してみるくらいでOKですがこのように短く書いても読みやすい(個人差ありなので任意)かも

```suggestion
        range.count ??= 0;
```

Reply:
修正しました！